mods/wlssb/random-teams.js

```diff
@@ -347,6 +346,22 @@ class RandomSeasonalRegStaffTeams extends RandomTeams {
 				},
 				nature: 'Modest',
 			},
```

Comment:
Mystifi is Admin again?

Reply:
%Lycanium Z: Wait Mystifi is admin again?
~HoeenHero: that was temp
%Lycanium Z: Oh